CSS/styles.css

```diff
@@ -66,12 +69,17 @@ p {
 }
 
 .aboutMe {
-  float: left;
-  display: inline-block;
+  left: 0;

```

Comment:
`left: 0;` doesn't seem to be doing anything, so this property can be removed.

Reply:
You're right. I've removed the `aboutMe` class